overlays/vault-pki-ha-overlay.yaml

```diff
@@ -0,0 +1,21 @@
+applications:
+  easyrsa: null
+  vault:
+    charm: cs:~openstack-charmers-next/vault
+    num_units: 2
+    options:
+      auto-generate-root-ca-cert: true
```

Comment:
Do we want to encourage this?

Reply:
I don't see any reason not to, as the default recommendation; it's equivalent to what EasyRSA does and identical to manually using the `generate-root-ca` action.  The only alternative is to run Vault as an intermediary CA which requires getting a cert signed by some other CA.